homeassistant/components/samsungtv/media_player.py

```diff
@@ -104,6 +115,10 @@ def __init__(
         self._config_entry = config_entry
         self._host: str | None = config_entry.data[CONF_HOST]
         self._mac: str | None = config_entry.data.get(CONF_MAC)
+        self._ssdp_location = (
+            config_entry.data.get(CONF_SSDP_RENDERING_CONTROL_LOCATION)
+            or f"http://{self._host}:9197/dmr"
+        )
```

Comment:
Should we drop the default value, and instead make SUPPORT_VOLUME_SET dependant on the presence of CONF_SSDP_RENDERING_CONTROL_LOCATION?

Reply:
done